src/Factory.php

```diff
@@ -19,12 +19,13 @@ public static function create(
         LoopInterface $loop,
         array $options = []
     ): Client {
+        $container = self::createContainer($loop, $options);
         return new Client(
```

Comment:
Actually returning the command bus instead of the client makes sense. Clients should using it building their client and abstract all operations into it :+1: .